tests/projects/AzureFunctionProject/Function1.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using System.Threading.Tasks;
+
+namespace StrykerReplication
+{
+    public class TestFunction
```

Comment:
The filename should be changed accordingly to the class name.

Reply:
done